ts/parts/Options.ts

```diff
@@ -3677,6 +3678,19 @@ H.defaultOptions = {
          * @apioption tooltip.split
          */
 
+        /**
+         * Prevents the tooltip from switching or closing, when touched or
+         * pointed.
+         *
+         * @sample highcharts/tooltip/stickonhover/
+         *         Tooltip sticks on hover event
+         *
+         * @type      {boolean}
+         * @since     8.0.1
+         * @product   highcharts highstock
```

Comment:
No support for Gantt and Highmaps?

Reply:
Good catch! I added support by removing the tag. :)